### ISSUE TITLE
Autocomplete plus highlighted row illegible

### ISSUE THREAD

**Issue by bruinebeer** [jesseweed/seti-ui#242]
Hi,

Autocomplete-plus & atom-spell-check in conjunction with seti-ui & seti-syntax dont work well together.
Some things have become illegible.

- autocomplete-plus:
  - highlighted row bg has same color as text
- spell-check
  - highlight row bg has same color as text
- less
  - dots are nearly (eg: .class-name) invisible when selected

I made some quick fixes by adding these less rules.

```Less
atom-text-editor {
  atom-overlay {
    .corrections {
      border: 4px solid black;
      border-radius: 3px;
      padding-top: 2px;
    }
    .list-group{
      background-color: black;
    }
    .autocomplete-suggestion-list .list-group .selected {
      background-color: #222;
      color: white;
    }
    .autocomplete-suggestion-list .list-group {
      background-color: black;
    }

    .corrections .list-group .selected {
      background-color: #222;
    }
    .corrections .list-group .selected::before,
    .corrections .list-tree .selected::before {
      background-color: rgba(0,0,0,0);
    }
  }
}
```

Keep up the good work.

**Comment by bruinebeer** [comment]
This css fixes all the issues for me:

```Less
atom-text-editor {
  atom-overlay {
    .corrections {
      border: 4px solid black;
      border-radius: 3px;
      padding-top: 2px;
    }
    .list-group{
      background-color: black;
    }
    .autocomplete-suggestion-list .list-group .selected {
      background-color: #222;
      color: white;
    }
    .autocomplete-suggestion-list .list-group {
      background-color: black;
    }

    .corrections .list-group .selected {
      background-color: #222;
    }
    .corrections .list-group .selected::before,
    .corrections .list-tree .selected::before {
      background-color: rgba(0,0,0,0);
    }
  }
}
```

**Comment by jesseweed** [comment]
This should be fixed in 1.2.1 update. Feel free to re-open is you still have an issue.
